web/Views/Landlord/Properties.cshtml

```diff
@@ -16,6 +16,7 @@
         <th scope="col">Created</th>
         <th scope="col">Rent (pcm)</th>
         <th scope="col">Description</th>
+        <th scope="col">View</th>
```

Comment:
I wouldn't actually put any text in this column, just an empty `<th></th>` looks nice (as we might put more actions in this column later, like Edit and Delete links.

Not your work but I also think the 'Created' column is unnecessary in this table, could we not show it?

Reply:
ok sure I'll delete it